CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @charly3pins @ramonmacias
```

Comment:
add newline at the end of the file :pray:

Reply:
Done